lib/routes/movie_details.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:flutter/material.dart';
+
+import '../repositories/implementation/genres_repository.dart';
+import '../repositories/implementation/movies_repository.dart';
+import '../repositories/interfaces/genres_repository_interface.dart';
+import '../repositories/interfaces/movies_repository_interface.dart';
+import '../ui_models/movie_preview.dart';
+import '../utils/constants.dart';
+import '../widgets/like_button.dart';
+import '../widgets/row_vote_average.dart';
+import '../widgets/scrollable_genres.dart';
+import '../widgets/text_info.dart';
+import '../widgets/text_subtitle.dart';
+import '../widgets/text_title.dart';
+
+class MovieDetails extends StatefulWidget {
+  const MovieDetails({super.key});
+
+  static const String routeName = "/movie";
+
+  @override
+  State<StatefulWidget> createState() => _MovieDetailsState();
+}
+
+class _MovieDetailsState extends State<MovieDetails> {
+  final MoviesRepositoryInterface movieRepository = MoviesRepository();
+  final GenresRepositoryInterface genresRepository = GenresRepository();
```

Comment:
Do you use these repositories here?

Reply:
No, I wrote this when i had just started with the module and wasn't sure if i needed them, but then  I forgot to delete them. Fixed :)